src/reducers/ingredientsReducer.js

```diff
@@ -0,0 +1,36 @@
+import {
+  GET_INGREDIENTS,
+  GET_INGREDIENTS_SUCCESS,
+  GET_INGREDIENTS_ERROR,
+} from '../actions/ingredients';
+
+const INITIAL_STATE = {
+  ingredients: [],
+};
```

Comment:
Como vc definiu o `loading` e o `error` que tal adicionar eles no `INITIAL_STATE`?
```suggestion
const INITIAL_STATE = {
  ingredients: [],
  loading: true,
  error: null,
};
```

Reply:
Verdade, ficou faltando definir o estado inicial. Obrigada! 😃